data/json/items/armor/suits_protection.json

```diff
@@ -843,10 +843,7 @@
     "symbol": "[",
     "looks_like": "touring_suit",
     "color": "dark_gray",
-    "covers": [ "LEGS", "TORSO", "ARMS" ],
-    "coverage": 95,
-    "encumbrance": 12,
-    "max_encumbrance": 25,
+    "coverage_data": [ { "parts": [ "torso", "leg_l", "leg_r", "arm_l", "arm_r" ], "coverage": 95, "encumbrance": [ 12, 25 ] } ],
```

Comment:
It looks like `ARMS` and `LEGS` should still work here.  Is that right?  If so, I suggest you use them.

Reply:
Good idea! I will use a mixture of old and new enum types to assure it works as intended.